app/main.py

```diff
@@ -2,24 +2,70 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: str) -> None:
```

Comment:
The `__get__` method should return the value of the attribute, not `None`. Ensure that the method returns the correct type, which should be `int` in this context.